proto/viam/component/motor/v1/motor.proto

```diff
@@ -43,6 +43,14 @@ service MotorService {
     };
   }
 
+  // SetRPM instructs the motor to move at the specified RPM indefinitely.
```

Comment:
lets remove the comment from `GoFor` that talks about this behavior

Reply:
I'll add a note that it's deprecated. in scope review yesterday we decided to leave the behavior as is for the next 3 ish months and mark it as deprecated in the meantime

Reply:
o sounds good!